README.rst

```diff
@@ -132,6 +132,7 @@ Edit your settings.py file:
         # Transform create of an existing Device (based on registration id) into
 		    # an update. See the section
         # "Update of device with duplicate registration ID" for more details.
+        # default: False
```

Comment:
Please amend in readme which django versions are still supported by this change

Reply:
btw, the `index.rst` file seems to not be exactly in sync with the `readme.rst`. Is there some kind of command to run to sync them or should I juste copy paste ?

Reply:
Just copy paste. It's not ideal I know :-)